src/Microsoft.VisualStudio.Services.Agent/ProcessChannel.cs

```diff
@@ -6,6 +6,8 @@
 
 namespace Microsoft.VisualStudio.Services.Agent
 {
+    public delegate void StartProcessDelegate(string pipeHandleOut, string pipeHandleIn);
```

Comment:
I renamed this from "ProcessStartDelegate" to "StartProcessDelegate". It seemed better for readability since delegates are effectively methods, and methods are named with verbs.

I'm guessing "ProcessStartDelegate" was originally chosen for consistency with "ProcessStartInfo". I would like to point out that System.Diagnostics.ProcessStartInfo is a class and therefore appropriately named using as a noun :)

Ultimately I felt like the readability was improved after switching the delegate to start with a verb.